test/Sema/object_literals_osx.swift

```diff
@@ -22,3 +22,5 @@ struct Path: _ExpressibleByFileReferenceLiteral {
 
 let p1: Path = #fileLiteral(resourceName: "what.txt")
 let p2 = #fileLiteral(resourceName: "what.txt") // expected-error{{could not infer type of file reference literal}} expected-note{{import Foundation to use 'URL' as the default file reference literal type}}
+
+let text = #fileLiteral(resourceName: "TextFile.txt").relativeString! // expected-error{{type of expression is ambiguous without more context}}
```

Comment:
It's because there's no `relativeString` and we're emitting a misleading diagnostic. Actually, I think the default `#fileLiteral` type isn't even around; we get a different diagnostic about to having a `relativeString` member with the real SDK